test/integration/resourcemanager/podzoneaffinity/podzoneaffinity_suite_test.go

```diff
@@ -157,11 +161,5 @@ func getMutatingWebhookConfigurations(namespace string) []*admissionregistration
 		},
 	}
 
-	webhookConfig[0].Webhooks[0].NamespaceSelector.MatchExpressions = append(webhookConfig[0].Webhooks[0].NamespaceSelector.MatchExpressions, metav1.LabelSelectorRequirement{
-		Key:      corev1.LabelMetadataName,
-		Operator: metav1.LabelSelectorOpIn,
-		Values:   []string{namespace},
-	})
```

Comment:
Please see PR description:
> Remove namespace name label selector from MutatingWebhookConfiguration since we already use another label selector which is sufficient. Also, the corev1.LabelMetadataName caused problems when stress testing.

If you use stress testing with a single `MutatingWebhookConfiguration` then you have to expand the namespace selector everytime a namespace is created which will result in conflicts. I think the other namespace selector `"control-plane.shoot.gardener.cloud/enforce-zone"` is already sufficient for our integration tests as it already makes sure that non-related namespaces are excluded.